crates/chia-consensus/src/gen/run_puzzle.rs

```diff
@@ -12,6 +13,7 @@ use clvmr::run_program::run_program;
 use clvmr::serde::{node_from_bytes, node_from_bytes_backrefs};
 use std::sync::Arc;
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
I think we should find a way to simplify this function somehow, but maybe it's not worth doing in this PR

Reply:
I agree. I suspect as we move the rust-python boundary forward, more opportunities will open up for that